frontend/model/settings/vuexModule.js

```diff
@@ -70,14 +70,16 @@ const mutations = {
     state.increasedContrast = isChecked
   },
   setNotificationEnabled (state, enabled) {
-    if (state.notificationEnabled !== enabled) {
+    if (state.notificationEnabled !== enabled && enabled) {
```

Comment:
Note: I might be both right and wrong about adding `&& enabled` here... investigating

- Right because it avoids infinite loop
- Wrong because it may be necessary to call `'service-worker/setup-push-subscription'` in both cases